packages/@react-aria/interactions/src/usePress.ts

```diff
@@ -907,3 +918,15 @@ function isValidInputKey(target: HTMLInputElement, key: string) {
     ? key === ' '
     : nonTextInputTypes.has(target.type);
 }
+
+function initEventCoordinates(originalEvent: Omit<EventBase, 'clientX' | 'clientY'>, currentTarget: Element):EventBase {
+  let x = 0;
+  let y = 0;
+
+  if (currentTarget instanceof HTMLElement) {
+    const rect = currentTarget.getBoundingClientRect();
+    x = rect.left + rect.width / 2;
+    y = rect.top + rect.height / 2;
```

Comment:
I think this should be
```
    x = rect.width / 2;
    y = rect.height / 2;
```
so that it's center of the target relative to the targets top/left

Reply:
This won't work unfortunately because the actual calculation happens in the constructor of PressEvent

Reply:
ah, you're correct, thank you for explaining